src/main/java/nb/kafka/operator/KafkaUtilities.java

```diff
@@ -82,19 +82,25 @@ public KafkaUtilities(String kafkaUrl, String securityProtocol, short defaultRep
     adminClient = AdminClient.create(conf);
     topics();
   }
-  
+
   public Set<String> topics() {
     KafkaFuture<Set<String>> names = adminClient.listTopics().names();
-    try {
-      KafkaFuture.allOf(names).get(10, TimeUnit.SECONDS);
-      Set<String> topics = names.get();
-      log.debug("Got topics: {}", topics);
-      return topics;
-    } catch (InterruptedException | ExecutionException | TimeoutException e) {
-      throw new IllegalStateException("Exception occured during topic retrieval.", e);
+    int count = 0;
+    int maxTries = 5;
+    while(true) {
+      try {
+        KafkaFuture.allOf(names).get(10, TimeUnit.SECONDS);
+        Set<String> topics = names.get();
+        log.debug("Got topics: {}", topics);
+        return topics;
+      } catch (InterruptedException | ExecutionException | TimeoutException e) {
+        if (++count == maxTries) {
```

Comment:
Maybe we should log something even when we don't throw exception, to know why we retry.